### PR TITLE
Update changelog for 0.3.0

### PR DESCRIPTION
**Pull request by ms-ati**
I'll rebase this after the "prepare for initial release" PR merges

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,12 +6,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+(nothing yet!)
+
+## [0.3.0] - 2021-02-10
```
Comment (jemmaissroff) [inline CHANGELOG.md:11]: Should we make this `2021-02-11`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -44,7 +48,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Panolint
 - Dependabot setup
 
-[Unreleased]: https://github.com/panorama-ed/memo_wise/compare/v0.2.0...HEAD
+[Unreleased]: https://github.com/panorama-ed/memo_wise/compare/v0.3.0...HEAD
+[0.2.0]: https://github.com/panorama-ed/memo_wise/compare/v0.2.0...v0.3.0
```
Comment (jemmaissroff) [inline CHANGELOG.md:52]: `[0.2.0]` --> `[0.3.0]`

**Review by jemmaissroff — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ms-ati) [reply CHANGELOG.md]: Updated!

File: CHANGELOG.md (hunk 2 above)
Comment (ms-ati) [reply CHANGELOG.md]: Oops good catch! Fixed!

**Review by jemmaissroff — APPROVED** [review]
LGTM once the first one merges!

**Pull request merged**